src/Pages/HomePage.jsx

```diff
@@ -27,215 +20,173 @@ const HomePage = () => {
+          <Text variant={"h3"} text={"Why Vacate?"}></Text>
           <Text
-            variant={"h5"}
+            variant={"h4"}
             text={
-              "Make it easier to find and be approved for rent or purchasing a home."
+              "A conviction vacation seals the offense from your record, and will give you more chance to access:"
             }
-            fontFamily={"Lora"}
           ></Text>
-        </div>
 
-        <div className="col-sm">
-          <img className="icons" src={employment} alt={"Employment Icon"} />
-          <Text variant={"h6"} text={"Employment"} fontWeight={"600"}></Text>
-          <Text
-            variant={"h5"}
-            text={"Reduce barriers to finding and obtaining employment."}
-            fontFamily={"Lora"}
-          ></Text>
-        </div>
+          <div className="col-sm">
+            <img className="icons" src={housing} alt={"Housing Icon"} />
+            <Text variant={"h6"} text={"Housing"} fontWeight={"600"}></Text>
+            <Text
+              variant={"h5"}
+              text={
+                "Make it easier to find and be approved for rent or purchasing a home."
+              }
+              fontFamily={"Lora"}
+            ></Text>
+          </div>
 
-        <div className="col-sm">
-          <img className="icons" src={education} alt={"Education Icon"} />
-          <Text variant={"h6"} text={"Education"} fontWeight={"600"}></Text>
-          <Text
-            variant={"h5"}
-            text={"Apply for scholarships, programs, degrees or certificates."}
-            fontFamily={"Lora"}
-          ></Text>
-        </div>
+          <div className="col-sm">
+            <img className="icons" src={employment} alt={"Employment Icon"} />
+            <Text variant={"h6"} text={"Employment"} fontWeight={"600"}></Text>
+            <Text
+              variant={"h5"}
+              text={"Reduce barriers to finding and obtaining employment."}
+              fontFamily={"Lora"}
+            ></Text>
+          </div>
 
-        <div className="col-sm">
-          <img className="icons" src={travel} alt={"Travel Icon"} />
-          <Text
-            variant={"h6"}
-            text={"Government Assistance"}
-            fontWeight={"600"}
-          ></Text>
-          <Text
-            variant={"h5"}
-            text={"Receive government help."}
-            fontFamily={"Lora"}
-          ></Text>
-        </div>
-        <div className="row">
-          <div className="col-lg">
-            {/* <SecondaryButton text={"Learn More"} /> */}
+          <div className="col-sm">
+            <img className="icons" src={education} alt={"Education Icon"} />
+            <Text variant={"h6"} text={"Education"} fontWeight={"600"}></Text>
+            <Text
+              variant={"h5"}
+              text={
+                "Apply for scholarships, programs, degrees or certificates."
+              }
+              fontFamily={"Lora"}
```

Comment:
`Lora` should be moved to a constant since it's used in many places and might need to be changed in the future.